packages/aws-cdk/lib/commands/flags.ts

```diff
@@ -0,0 +1,37 @@
+import type { FeatureFlagReportProperties } from '@aws-cdk/cloud-assembly-schema';
+import { info } from '../logging';
+
+export async function displayFlags(flagsData: FeatureFlagReportProperties[]): Promise<void> {
+  info('Feature Flags Report:');
+  const headers = ['Feature Flag Name', 'Recommended Value', 'User Value'];
+
+  for (const report of flagsData) {
+    const flags = Object.entries(report.flags);
+    const columnWidths = [
+      Math.max(headers[0].length, ...flags.map(([name]) => name.length)),
+      Math.max(headers[1].length, ...flags.map(([, flag]) => String(flag.recommendedValue).length)),
+      Math.max(headers[2].length, ...flags.map(([, flag]) => String(flag.userValue).length)),
+    ];
```

Comment:
Good stuff, but this should probably be in a helper function.

And while you're writing that, search the code base because it would surprise me if we didn't have something like this already.